webviz_subsurface/plugins/_reservoir_simulation_timeseries.py

```diff
@@ -157,11 +182,23 @@ def __init__(
         column_keys: list = None,
         sampling: str = "monthly",
         options: dict = None,
+        predefined_expressions: str = None,
         line_shape_fallback: str = "linear",
     ):
 
         super().__init__()
 
+        WEBVIZ_ASSETS.add(
+            Path(webviz_subsurface.__file__).parent
+            / "_assets"
+            / "css"
+            / "block_options.css"
+        )
+
+        WEBVIZ_ASSETS.add(
+            Path(webviz_subsurface.__file__).parent / "_assets" / "css" / "modal.css"
+        )
```

Comment:
These are CSS styles not supported yet in `wcc`? If so, should we make some issue in that repository on generalizing them, or is this very specialized to the Vector Calculator?

Reply:
Temporary css. Pending on new wcc modal component.

Issue created for modal component: https://github.com/equinor/webviz-core-components/issues/163